### DISCUSSION TITLE
Update map easily

### DISCUSSION THREAD

**Discussion by Starmania** [organicmaps/organicmaps#1146] [Ideas]
Hey team,

All is in the title, but I will explain some feature that's had with.

> Toast notification: 
    - Just for saying to the user an update of a region of the map was finished.

> Notification alert:
    - If user disable auto-update of map, saying "Hey, there is update for this zone"
    - And for download progress

> Last update:
    - Just add a date for the last update of region, also we will see if a update was downloaded recently.

It's all... And good luck if you add feature!

PS: I'm french, also m'y english is not too good...

![Screenshot_2021-08-21-18-34-23-461_app organicmaps](https://user-images.githubusercontent.com/52313983/130328719-1d449e03-27f1-485d-a59b-f6cbf85dd5aa.jpg)

**Comment by biodranik** [comment]
Hi and thanks for the suggestions! We'll try to improve this feature. Usually a red dot should appear on top of the menu button indicating that updates are available.
Current maps data version is displayed in Settings / About.
